src/main/scala/mesosphere/mesos/LibMesos.scala

```diff
@@ -8,6 +8,15 @@ object LibMesos extends StrictLogging {
 
   /**
     * The changelog should describe why this version is needed.
+    * 
+    * **ATTENTATION**
+    * Make sure that the latest Marathon supports latest Mesos -2 versions.
+    * Eg Marathon 1.10 should support Mesos 1.10, 1.9 and 1.8. This ensures
+    * that DC/OS upgrades won't break. If you must violate this rule be sure
+    * to visit the DC/OS upgrade procedure with other engineers.
```

Comment:
wouldn't it makes sense to mentions that skip minor version upgrades should be avoided?

Reply:
I'm not sure I understand. This comment is for Marathon developers. Operators should follow the [instructions](https://mesosphere.github.io/marathon/docs/upgrade/index.html#releases) which state that

> It is recommended not to skip Marathon releases when upgrading Marathon. For instance, if upgrading to 1.6.x from 1.4.x, you should upgrade to 1.5.x first, and only then to 1.6.x.

Reply:
👍